cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,283 @@
+                    "caps": [
+                        "mon", "allow r",
+                              "mgr", "allow rw",
+                              "osd", "allow rw tag cephfs *=*",
+                              "mds", "allow rw"],
+                    "format": "json"}
+        ret_val, json_out = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-cephfs-node' command failed")
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-cephfs-node' key")
+
+    def create_cephCSIKeyring_RBDProvisioner(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": "client.csi-rbd-provisioner",
+                    "caps": ["mon", "profile rbd",
+                             "mgr", "allow rw",
+                             "osd", "profile rbd"],
+                    "format": "json"}
+        ret_val, json_out = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-rbd-provisioner' command failed")
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-rbd-provisioner' key")
+
+    def create_cephCSIKeyring_RBDNode(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": "client.csi-rbd-node",
+                    "caps": ["mon", "profile rbd",
+                             "osd", "profile rbd"],
+                    "format": "json"}
+        ret_val, json_out = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-rbd-node' command failed")
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-rbd-node' key")
+
+    def create_checkerKey(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": self.run_as_user,
+                    "caps": ["mon", "allow r, allow command quorum_status"],
+                    "format": "json"}
+        ret_val, json_out = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create {}' command failed".format(self.run_as_user))
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-rbd-node' key")
```

Comment:
this need to be changed, still referring to csi-rbd-node

Reply:
@Madhu-1 , what is the change we are expecting here?

Reply:
earlier in some function which is not related to csi-rbd node log was added. as the commit is overwritten when you force push I cannot find the reference anymore